cpp/bench/CMakeLists.txt

```diff
@@ -54,6 +54,18 @@ if(BUILD_CUML_BENCH)
     PRIVATE
       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../src_prims>
   )
+
+  set_target_properties(
```

Comment:
Drop installing the benchmarks for now, we can do that in a different PR so it is consitent across all of RAPIDS